checks/system/ce/ce_import_run_image.py

```diff
@@ -36,7 +36,7 @@ class enroot_import_image_dockerhub(enroot_import_image):
 
 class enroot_import_image_ngc(enroot_import_image):
     image = variable(
-        str, value='docker://nvcr.io#nvidia/k8s/cuda-sample:nbody'
+        str, value='docker://nvcr.io#nvidia/hpc-benchmarks:24.03'
```

Comment:
missing space line 22

Reply:
+  lines 47 and 67

Reply:
> * lines 47 and 67

What's the problem there?

Reply:
- pycodestyle checks/system/ce/ce_import_run_image.py
```
checks/system/ce/ce_import_run_image.py:22:24: E225 missing whitespace around operator
checks/system/ce/ce_import_run_image.py:47:25: E261 at least two spaces before inline comment
checks/system/ce/ce_import_run_image.py:67:25: E261 at least two spaces before inline comment
```